src/v0/sources/shopify/transform.js

```diff
@@ -126,6 +129,25 @@ const processEvent = (inputEvent) => {
     case ECOM_TOPICS.CHECKOUTS_UPDATE:
       message = ecomPayloadBuilder(event, shopifyTopic);
       break;
+    case 'carts_create':
+    case 'carts_update':
+      if (checkForValidRecord(inputEvent)) {
+        message = trackPayloadBuilder(event, shopifyTopic);
+      } else {
+        /**
+         *  This Scenario handles the case when empty cart events are passed
```

Comment:
Discarding events depending on empty cart event is not seems correct to me. It may be possible for someone removes all the item from the cart and then also we get the empty cart.

Reply:
Implementation is already updated on this note but forgot to update the commenting.

Reply:
Done